client/my-sites/plugins/main.jsx

```diff
@@ -220,6 +221,10 @@ export default React.createClass( {
 		}
 	},
 
+	goToBrowser() {
```

Comment:
it's the method that executes just before changing the url. I'll change it to `onBrowserLinkClick" to make it clearer